src/main/main.ts

```diff
@@ -115,9 +133,35 @@ app
   .catch(Logger.error);
 
 eventBus.on('USER_LOGGED_IN', async () => {
-  await AppDataSource.initialize();
+  if (!AppDataSource.isInitialized) {
+    await AppDataSource.initialize();
+  }
+  getAuthWindow()?.destroy();
+  await createWidget();
+  const widget = getWidget();
+  const tray = getTray();
+  if (widget && tray) {
+    setBoundsOfWidgetByPath(widget, tray);
+  }
+
+  const lastOnboardingShown = configStore.get('lastOnboardingShown');
+
+  if (!lastOnboardingShown) {
+    openOnboardingWindow();
+  } else if (widget) {
```

Comment:
This will be not enough to avoid displaying the widget before the onboarding as when the widget is ready it bee shown and the auto-launch is not enable it's shown

Reply:
I removed the widget.show when widget is ready since we open it imperatively now